python/ray/tests/test_gcs_fault_tolerance.py

```diff
@@ -828,6 +829,41 @@ def f():
     wait_for_pid_to_exit(gcs_server_pid, 10000)
 
 
+@pytest.mark.parametrize(
+    "ray_start_regular",
```

Comment:
It will run in the "External redis tests" unless there is a pytest.skipif. An earlier commit had the skipif instead of the alive check for Redis, and an even earlier commit failed the Redis check.